### ISSUE TITLE
Image Gallery Support

### ISSUE THREAD

**Issue by sfreytag** [sfreytag/friday-theme#3]
This would be nice for a blog.

**Comment by sfreytag** [comment]
For anyone looking for an image gallery layout, check out the great use of Boostrap's card-columns here:

https://selimamrouni.github.io/index.html

**Comment by sfreytag** [comment]
The docs already have an example of a Masonry layout. Now that I have upgraded to 4.4, I have added two more:

A carousel:

https://sfreytag.github.io/friday-theme/2020/03/02/friday-theme-carousel.html

And Masonry with a lightbox:

https://sfreytag.github.io/friday-theme/2020/03/01/friday-theme-lightbox.html

I think this fixes this image gallery job. The technique is the same for all of them:

1. List the images in front matter
2. Use Liquid to build an HTML structure
3. Add JS to convert it to a gallery

And for (2) and (3) there's some handy JS in the Bootstrap components, which the two blog posts above explain how to use.

If anyone uses this, let me know how you get on!
